app/src/main/java/io/stanwood/bitrise/ui/builds/vm/BuildsViewModel.kt

```diff
@@ -53,6 +54,16 @@ class BuildsViewModel(private val router: NavController,
     val title: String
         get() = app.title
 
+    val isProvidedByGithub: Boolean
```

Comment:
Do you prefer using this as opposed to `fun isProvidedByGithub() = app.provider == RepoProvider.GITHUB`? If so, why? Just being curious.

Reply:
If a method/function has no argument and returns something else than Unit I always replace it with property